plover/oslayer/xkeyboardcontrol.py

```diff
@@ -1214,6 +1219,9 @@ def send_backspaces(self, number_of_backspaces):
         for x in range(number_of_backspaces):
             self._send_keycode(self._backspace_mapping.keycode,
                                self._backspace_mapping.modifiers)
+            if self._time_between_key_presses != 0:
+                sleep(self._time_between_key_presses / 1000)
+                self._display.sync()
```

Comment:
Perhaps the sync should be before the sleep, for consistency with other platforms.

(usually the delay is small and the user won't notice this anyway?)